src/publishPackages/publish.proj

```diff
@@ -53,33 +56,84 @@
 
   <Target Name="PreparePublish" DependsOnTargets="$(CreateDependsOn)">
     <MakeDir Directories="$(ManifestDir)" />
+    <MakeDir Directories="$(ArtifactsTempWorkDir)" />
     <PropertyGroup>
       <ExpectedFeedUrl>https://$(AzureAccountName).blob.core.windows.net/$(ContainerName)/index.json</ExpectedFeedUrl>
```

Comment:
The ExpectedFeedUrl does end up in the Manifest. However, due to [arcade changes](https://github.com/dotnet/arcade/blob/9b3f304c7bc9fd4d11be9ca0b466b83e98d2a191/Documentation/CorePackages/Publishing.md#basic-onboarding-scenario) the actual feed we end up utilizing is based off of `_DotNetArtifactsCategory` in [SetupTargetFeeds](https://github.com/dotnet/arcade/blob/fd91e27589e69c0a97db2e208b112a24ab989180/src/Microsoft.DotNet.Arcade.Sdk/tools/SdkTasks/SetupTargetFeeds.proj#L101)